CHANGELOG.md

```diff
@@ -1,5 +1,243 @@
 # Changelog
 
+## v0.66.0
+
+## v0.66.0-rc.0
+
+### Breaking
+
+- Bump Hermes npm to 0.8.0 ([2b67631210](https://github.com/facebook/react-native/commit/2b67631210070de0c07b21c69fb9d0ccd9e9fcba) by [@Huxpro](https://github.com/Huxpro))
```

Comment:
note: we want a more up-to-date version, so probably will need to be 0.8.1

Reply:
Hm odd, I thought we were on 0.8.1 for 0.65 already.

Reply:
Edit: we're planning for 0.9.0 for this

Reply:
These show up because the changes to bump Hermes were made after the initial 0.65 cut. So the changes are in the 0.65 branch, but showed up as separate commits in the main branch later.

Reply:
yeah it's weird that this is showing up, on 0.65 there's already Hermes 0.8.1 - probably just the generator acting up

Reply:
The same issue affects RN CLI which has been bumped to `6.0.0` in `0.65.0-rc.3`, but the commit message appears in this changelog.